src/styles/index.ts

```diff
@@ -3615,7 +3615,6 @@ const styles = (theme: ThemeColors) =>
         },
 
         searchInputStyle: {
-            color: theme.textSupporting,
             fontSize: 13,
```

Comment:
NAB: This could technically be one of our font size variables

Reply:
Actually I wonder why this is 13px and not our standard font size of 15px? cc @Expensify/design for thoughts

Reply:
I updated the PR to use 15px. Let me know what you think

Reply:
Love it. And as discussed, let's make sure none of these styles impact other search inputs across the app just yet.